dotnet-desktop-guide/net/wpf/properties/how-to-override-metadata-for-a-dependency-property.md

```diff
@@ -0,0 +1,40 @@
+
+A derived class that overrides metadata on a dependency property must do so before the property is placed in use by the property system. A dependency property is placed in use when any instance of the class that registers the property is instantiated. To help meet this requirement, the derived class should call <xref:System.Windows.DependencyProperty.OverrideMetadata%2A> within its static constructor. Overriding the metadata of a dependency property after its owner type is instantiated won't raise exceptions, but will result in inconsistent behaviors in the property system. Also, a derived type can't override the metadata of a dependency property more than once, and attempts to do so will raise an exception.
+
+## Example
+
+In the following example, the derived class `TropicalAquarium` overrides the metadata of a dependency property inherited from the base class `Aquarium`. The metadata type is <xref:System.Windows.FrameworkPropertyMetadata>, which supports UI-related WPF framework characteristics such as <xref:System.Windows.FrameworkPropertyMetadataOptions.AffectsRender>. The derived class doesn't override the inherited `AffectsRender` flag, but it does update the default value of `AquariumGraphic` on derived class instances.
+
+:::code language="csharp" source="./snippets/how-to-override-metadata-for-a-dependency-property/csharp/MainWindow.xaml.cs" id="BaseDependencyProperty":::
+:::code language="vb" source="./snippets/how-to-override-metadata-for-a-dependency-property/vb/MainWindow.xaml.vb" id="BaseDependencyProperty":::
+:::code language="csharp" source="./snippets/how-to-override-metadata-for-a-dependency-property/csharp/MainWindow.xaml.cs" id="InheritedDependencyProperty":::
+:::code language="vb" source="./snippets/how-to-override-metadata-for-a-dependency-property/vb/MainWindow.xaml.vb" id="InheritedDependencyProperty":::
+
+## See also
+
+- <xref:System.Windows.DependencyProperty>
+- [Dependency property metadata](dependency-property-metadata.md)
+- [Dependency properties overview](dependency-properties-overview.md)
+- [Custom dependency properties](custom-dependency-properties.md)
+- [How-to topics](/dotnet/desktop/wpf/advanced/properties-how-to-topics?view=netframeworkdesktop-4.8&preserve-view=true)
```

Comment:
we don't need these old link farm TOC articles. If anything, we should update the overview article with links to the new howtos.
```suggestion
```